packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.ts

```diff
@@ -132,9 +132,15 @@ export class ChromesHandler extends RenderMiddlewareBase {
         }
       }
       requestUrl.searchParams.append('timestamp', Date.now().toString());
+
+      const normalizedHeaders: HeadersInit = Object.entries(headers).reduce((acc, [key, value]) => {
```

Comment:
Would it be better to normalize them in `getHeadersForPropagation` ?

Reply:
make sense.